runtime-attach/runtime-attach-core/build.gradle.kts

```diff
@@ -4,6 +4,7 @@ plugins {
 }
 
 description = "To help in create an OpenTelemetry distro able to runtime attach an OpenTelemetry Java Instrumentation agent"
+otelJava.moduleName.set("io.opentelemetry.contrib.attach")
```

Comment:
we could put this under `io.opentelemetry.contrib.attach.core` to not conflict with below

Reply:
👍